### ISSUE TITLE
How to set parameters for tapas-base when fine-tunning?

### ISSUE THREAD

**Issue by Facico** [NielsRogge/Transformers-Tutorials#51]
Hi, nice tutorials!

Thank you for adding TAPAS to transformers. It is really helpful.

However, when I finetune this model in SQA all data, I could not get the accuracy 0.6737 computer by tapas-base-finetuned-sqa.( I set the batch size to 2 as the Fine_tunning_TapasForQuestion_on_SWA.ipynb )
And the parameters from paper are for tapas-large, I don't think it should be applicable to be the fine-tunning of tapas-base. How can I fine-tune correctly？

Thank you！

**Comment by NielsRogge** [comment]
Hi, thanks for your message!

Note that the "fine-tuning TapasForQuestionAnswering on SQA" notebook was mainly for demonstration purposes, things like batch size, learning rate etc. might be different in the original paper.

I wonder what the accuracy is that you end up with?

**Comment by Facico** [comment]
Just about 0.5 in test data（and the accuracy is about 0.62 in train data）.

**Comment by Facico** [comment]
The hyper parameters of tapas-base can be seen in this [issue](https://github.com/google-research/tapas/issues/145)
